kolibri/core/webpack/hooks.py

```diff
@@ -158,11 +159,7 @@ def build_path(self):
         An auto-generated path to where the build-time files are stored,
         containing information about the built bundles.
         """
-        return os.path.join(
-            os.path.abspath(os.path.dirname(__name__)),
-            self._module_file_path,
-            "build"
-        )
+        return resource_filename('kolibri.core', 'build')
```

Comment:
We use `resource_filename` to allow our assets to be discovered even when installed in a different location (it will use the package directory vs. your current directory).